docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -114,14 +114,66 @@ Complete list of `SQL Server data types
 
 .. include:: jdbc-type-mapping.fragment
 
+.. _sqlserver-sql-support:
+
 SQL support
 -----------
 
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/delete`
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
+The connector provides **read access and limited write access** to data and
+metadata in SQL Server. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+  No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/select` including:
+
+ * :doc:`/sql/match-recognize`
+ * :doc:`/sql/values`
+
+* :doc:`/sql/show-catalogs`
+* :doc:`/sql/show-columns`
+* :doc:`/sql/show-schemas`
+* :doc:`/sql/show-tables`
+* :doc:`/sql/describe`
+
+Write access:
+
+* :doc:`/sql/create-schema`
+* :doc:`/sql/create-table`
+* :doc:`/sql/create-table-as`
+* :doc:`/sql/drop-schema`
+* :doc:`/sql/drop-table`
+* :doc:`/sql/insert`
+* :doc:`/sql/show-create-schema`
+* :doc:`/sql/show-create-table`
+* :doc:`/sql/update`
+
+Security:
```

Comment:
I dont think the connector supports any security stuff

Reply:
Correct. Roles are only fully supported in the Hive connector AFAIK. But please check with Dain maybe.